### PR TITLE
feat: Add basic feature flag page

### PR DESCRIPTION
**Pull request by tomquirk**
Super light weight page for feature flags. Better than the console.

### CODE REVIEW

File: src/pages/flags/index.page.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { JuiceSwitch } from 'components/JuiceSwitch'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+import { readNetwork } from 'constants/networks'
+import { useCallback, useState } from 'react'
+import { featureFlagEnabled, setFeatureFlag } from 'utils/featureFlags'
+import Head from 'next/head'
+
+export default function FlagsPage() {
+  const [, updateState] = useState({})
+  const forceUpdate = useCallback(() => updateState({}), [])
```
Comment (tomquirk) [inline src/pages/flags/index.page.tsx:11]: i have sinned

Comment (wraeth-eth) [reply src/pages/flags/index.page.tsx:11]: explain

Comment (tomquirk) [reply src/pages/flags/index.page.tsx:11]: @wraeth-eth component doesnt re-render when localstorage is updated, so need to trigger a re-render somehow. Feels hacky but idk, probably fine.

Comment (wraeth-eth) [reply src/pages/flags/index.page.tsx:11]: oh lol - I see. Sounds good. Maybe a better way, but not worth slaving over

**Pull request merged**